.github/workflows/build-and-test.yaml

```diff
@@ -27,6 +27,11 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v3
 
+      - name: Free disk space (Ubuntu)
```

Comment:
@shmpwk You need to update this part as well.
https://github.com/autowarefoundation/autoware.universe/blob/e0187723d17dead16a441812af5a4dd4ab04354d/.github/sync-files.yaml#L34-L44

Maybe adding this before `Check out repository` is `sd`-friendly.

Reply:
Thank you for your review.
Instead of changing this sync setting, I'd like to add this PR feature to autoware_common.
https://github.com/autowarefoundation/autoware_common/pull/197
Do you agree this solution?

Reply:
@kenji-miyake

Reply:
> Instead of changing this sync setting, I'd like to add this PR feature to autoware_common.

@shmpwk If you want to apply it to all synced repositories, then I think it's fine. I'll leave it to you.

Reply:
Thank you. I agree that this change can be applied to other repositories.